synapse/models/infotech.py

```diff
@@ -693,7 +693,7 @@ def getModelDefs(self):
                     ('url', ('inet:url', {}), {
                         'doc': 'A URL linking this CVE to a full description.',
                     }),
-                    ('references', ('array', {'type': 'inet:url', 'uniq': True}), {
+                    ('references', ('array', {'type': 'inet:url', 'uniq': True, 'sorted': True}), {
```

Comment:
This is not migrated (it is already uniqued) but not sorted in storage.

Reply:
Yep, we should probably migrate these to be ordered in storage